recipe/meta.yaml

```diff
@@ -23,27 +23,28 @@ requirements:
     - python >=3.9
     - versioningit
   run:
+    - cinnabar <0.5
     - click
     - gufe >=1.0.0rc0
+    - kartograf >=1.0.0
     - lomap2 >=2.1.0
     - networkx
     - numpy
+    - openfe-analysis >=0.2.0
+    - openff-models ~=0.1.2
```

Comment:
since we essentially inherit gufes pins, can we leave this pin out?

Reply:
Just `openff-models` right?